components/eamxx/src/control/atmosphere_driver.cpp

```diff
@@ -243,7 +257,7 @@ void AtmosphereDriver::create_grids()
   // To avoid having to pass the same data twice in the input file,
   // we have the AD add the IC file name to the GM params
   const auto& ic_pl = m_atm_params.sublist("initial_conditions");
-  if (m_case_t0<m_run_t0) {
+  if (m_run_type==RunType::Restart) {
```

Comment:
These checks are equivalent, but I thought using the enum may be more robust if we later decide to distinguish Continue from Branch, or maybe support Hybrid (which I still don't understand).